moped-editor/src/views/projects/projectView/ProjectComponentEdit.js

```diff
@@ -1016,11 +834,6 @@ const ProjectComponentEdit = ({
           </>
         }
       />
-      {error && (
-        <Alert className={classes.mapAlert} severity="error">
-          {mapErrors.failedToSave}
-        </Alert>
-      )}
```

Comment:
this `error` was actually tied to the `COMPONENT_DETAILS_QUERY` that i've deprecated. as far as i can tell we have not actually been handling component mutation errors anywhere.